src/validation.cpp

```diff
@@ -1851,6 +1851,18 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_NULLDUMMY;
     }
 
+    // SEGWIT2X signalling.
+    if ( VersionBitsState(pindex->pprev, chainparams.GetConsensus(), Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE &&
```

Comment:
From [an exchange regarding BIP91 on the devlist](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-May/014397.html).

> I didn't think a lock in period was all that good an idea here due to the fairly short deployment timeline.

It seems that a lack of LOCKED_IN was the original intent behind BIP91 so a lack of bypass in the code should be regarded as a bug.